.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,10 @@ Make sure that the following steps are done before merging:
     - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
     - [ ] Add an entry to the CHANGELOG.
   - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?
+
+If changes are made as part of the base installation, then you need to issue a pull requests to all the basic branches:
```

Comment:
IMHO it would be cool to add a link that automatically creates a pull request like (Create PR at: https://github.com/edx/configuration/compare/master...raccoongang:ginkgo-rg)

Reply:
there is no easy way to automatically prepare such PRs because release branches mostly incompatible and will generate many conflicts when compared